src/System.IO.Ports/tests/Configurations.props

```diff
@@ -1,10 +1,11 @@
 ﻿<Project>
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard-Windows_NT;
-      netstandard-Linux;
-      netstandard-OSX;
+      netcoreapp-Windows_NT;
+      netcoreapp-Linux;
```

Comment:
This could be simplified as the csproj could just use `TargetsWindows == true` instead of `$(TargetsLinux)' != 'true' AND '$(TargetsOSX)' != 'true'`